### PR TITLE
chore(linux): Remove legacy projects

### PR DESCRIPTION
**Pull request by ermshiperete**
This change removes the KMFL legacy projects. They are still available in the stable-16 branch if we need them.

Also removed some shellcheck warnings.

@keymanapp-test-bot skip

### CODE REVIEW

File: linux/ibus-keyman/src/keymanutil.h [hunk 1]
```diff
@@ -3,9 +3,9 @@
 /*
  * Keyman Input Method for IBUS (The Input Bus)
  *
- * Copyright (C) 2018, 2020 SIL International
+ * Copyright (C) 2018-2023 SIL International
  *
- * kmflutil is dual licensed under the MIT or GPL licenses as described below.
+ * keymanutil is dual licensed under the MIT or GPL licenses as described below.
```
Comment (darcywong00) [inline linux/ibus-keyman/src/keymanutil.h:8]: If KMFL is removed, does this mean we only  use the MIT license now?

**Comment by darcywong00** [comment]
Does this address part of #6189?

**Comment by ermshiperete** [comment]
> Does this address part of #6189?

Yes, this makes #6189 obsolete. Thanks for the reminder, I had intended to link that one but then forgot.

**Review by mcdurdin — APPROVED** [review]

File: linux/ibus-keyman/src/keymanutil.h (hunk 1 above)
Comment (ermshiperete) [reply linux/ibus-keyman/src/keymanutil.h:8]: No, there are still files under GPL.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.37-alpha](https://keyman.com/downloads/releases/alpha/17.0.37)

**Pull request merged**